io/src/main/scala/fs2/io/package.scala

```diff
@@ -0,0 +1,104 @@
+package fs2
+
+import fs2.util.{Async, Effect}
+import fs2.util.syntax._
+import java.io.{InputStream, OutputStream}
+
+package object io {
+
+  private[io] def readBytesFromInputStream[F[_]](is: InputStream, buf: Array[Byte])(implicit F: Effect[F]): F[Option[Chunk[Byte]]] =
```

Comment:
just merely style guide. How about introducing JavaInputOutputStream.scala, put all implementation there and on package object just have the `api` functions?